packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -36,14 +36,25 @@ class HorizontalOverflow extends Foundation<
         scrollDuration: void 0,
         managedClasses: void 0,
         onScrollChange: void 0,
+        onHorizontalOverflowChange: void 0,
     };
 
     private horizontalOverflowItemsRef: React.RefObject<HTMLUListElement>;
 
     /**
-     * Throttle request animation frame usage
+     * Throttle scroll request animation frame usage
      */
-    private throttled: any;
+    private throttledScroll: throttle;
+
+    /**
+     * Throttle resize request animation frame usage
+     */
+    private throttledResize: throttle;
+
+    /**
+     * The scroll change object (so it can be accessed in other callbacks)
+     */
+    private scrollChangeObject: ScrollChange = {} as ScrollChange;
```

Comment:
Why are you setting this as an empty object and casting it? Do we need to set it as an empty object here?